### PR TITLE
added About App screen

### PR DESCRIPTION
**Pull request by antropovAlexandr**
added About App screen

### CODE REVIEW

File: src/navigation/navigators/AboutAppNavigator.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          />
+        ),
+        title: translate("aboutApp.aboutApp"),
+        ...headerStyles
+      };
+    }
+  }
+);
+
+AboutAppStackNavigator.navigationOptions = () => ({
```
Comment (KunskayaV) [inline src/navigation/navigators/AboutAppNavigator.js]: u can apply navigationOptions directly inside the function call above

File: src/screens/AboutApp/styles.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import { StyleSheet } from "react-native";
+import { black87, black06, blue, gray } from "../../constants/colors";
+
+const styles = StyleSheet.create({
+  container: {
+    margin: 20
```
Comment (KunskayaV) [inline src/screens/AboutApp/styles.js]: it's rather strange, that content hides earlier on the top than reaches header. the same for the bottom
Screen Shot 2019-12-23 at 9 45 45 PM

File: src/screens/AboutApp/styles.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    fontSize: 12,
+    lineHeight: 16,
+    letterSpacing: 2,
+    textTransform: "uppercase"
+  },
+  contactDetailsItem: {
+    marginTop: 10,
+    paddingBottom: 10,
+    borderBottomColor: gray,
+    borderBottomWidth: 1
```
Comment (KunskayaV) [inline src/screens/AboutApp/styles.js]: border on the mockup touches the right edge. no margin between

File: src/screens/AboutApp/styles.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+    letterSpacing: 0.15
+  },
+  contactDetailsInput: {
+    color: black06,
+    fontFamily: "Roboto",
+    fontSize: 14,
+    lineHeight: 20,
+    letterSpacing: 0.25
+  }
+});
```
Comment (KunskayaV) [inline src/screens/AboutApp/styles.js]: colors and border width are not equal to mockup's

File: src/screens/AboutApp/AboutApp.js [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+        {`${translate("aboutApp.version")} ${version}`}
+      </Text>
+    </View>
+    <Text style={styles.text}>{translate("aboutApp.description")}</Text>
+    <Button
+      caption={translate("aboutApp.readMore")}
+      appearance="Light"
+      wrapperStyles={styles.buttonWrapper}
+      borderColorStyle={styles.signUpBorder}
+      onPress={() => {}}
```
Comment (KunskayaV) [inline src/screens/AboutApp/AboutApp.js]: I think some link should be provided to be opened

File: src/screens/AboutApp/styles.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+  version: {
+    fontSize: 12
+  },
+  buttonWrapper: {
+    marginTop: 20
+  },
+  text: {
+    marginTop: 10,
+    color: black87,
+    fontFamily: "Roboto",
```
Comment (KunskayaV) [inline src/screens/AboutApp/styles.js]: why do we use different fonts on the same screen? is it on purpose or just designer mistake?

File: src/navigation/navigators/AboutAppNavigator.js (hunk 1 above)
Comment (antropovAlexandr) [reply src/navigation/navigators/AboutAppNavigator.js]: done

File: src/screens/AboutApp/styles.js (hunk 2 above)
Comment (antropovAlexandr) [reply src/screens/AboutApp/styles.js]: fixed

File: src/screens/AboutApp/styles.js (hunk 3 above)
Comment (antropovAlexandr) [reply src/screens/AboutApp/styles.js]: done

File: src/screens/AboutApp/AboutApp.js (hunk 5 above)
Comment (antropovAlexandr) [reply src/screens/AboutApp/AboutApp.js]: its will be fixed when we will know about  that

File: src/screens/AboutApp/styles.js (hunk 6 above)
Comment (antropovAlexandr) [reply src/screens/AboutApp/styles.js]: designer use two different fonts in our layout

File: src/screens/AboutApp/AboutApp.tsx [hunk 7]
```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { ScrollView, View, Text, Image } from "react-native";
+import { translate } from "../../i18n";
+import { Button } from "../../components";
+
+import styles from "./styles";
+
+const logo = require("../../assets/logotype/Logo.png");
```
Comment (vkunskaya) [inline src/screens/AboutApp/AboutApp.tsx]: U renamed 'logo.png' to 'Logo.png'. But other screens still require 'logo.png' and therefore we have packager issue. Pls, fix imports in all files.

Screen Shot 2020-01-08 at 10 26 53 PM

File: src/screens/AboutApp/AboutApp.tsx [hunk 8]
```diff
@@ -0,0 +1,80 @@
...
+        <Text style={styles.title}>
+          {translate("aboutApp.developmentTitle")}
+        </Text>
+        <Text style={styles.text}>
+          {translate("aboutApp.developmentDescription")}
+        </Text>
+      </View>
+      <Image resizeMode="contain" style={styles.epamLogo} source={epamLogo}/>
+    </View>
+  </ScrollView>
```
Comment (vkunskaya) [inline src/screens/AboutApp/AboutApp.tsx:70]: right text padding
Screen Shot 2020-01-08 at 10 28 12 PM

Comment (vkunskaya) [reply src/screens/AboutApp/AboutApp.tsx:70]: and pay attention on text: "Идея пр**ии**ложения...". Too many commas as well, e.g. "После победы в хакатоне, приложение взяли разработчики, в рамках практики на учебном курсе FrontSpot Hub, внутри компании EPAM." Punctuation should be checked.

File: src/screens/AboutApp/AboutApp.tsx [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+      <Button
+        caption={translate("aboutApp.readMore")}
+        appearance="Light"
+        wrapperStyles={styles.buttonWrapper}
+        onPress={() => {
+          // TODO add logic when we will know what exactly should do current button
+        }}
+      />
+      <View style={styles.article}>
+        <Text style={styles.title}>{translate("aboutApp.aboutCenter")}</Text>
```
Comment (vkunskaya) [inline src/screens/AboutApp/AboutApp.tsx:36]: Font in the app is not the same as in invision. And I don't see Lato font in assets

Screen Shot 2020-01-08 at 10 36 05 PM

Comment (antropovAlexandr) [reply src/screens/AboutApp/AboutApp.tsx:36]: Fonts will be add in another PR

File: src/screens/AboutApp/AboutApp.tsx (hunk 7 above)
Comment (antropovAlexandr) [reply src/screens/AboutApp/AboutApp.tsx]: fixed

File: src/screens/AboutApp/AboutApp.tsx (hunk 8 above)
Comment (antropovAlexandr) [reply src/screens/AboutApp/AboutApp.tsx:70]: done

**Pull request merged**